templates/default/opencast-setenv.erb

```diff
@@ -42,6 +42,7 @@ NEWRELIC="-javaagent:${NEWRELIC_JAR} $NEWRELIC_CONFIG"
 NEWRELIC=""
 <% end %>
 
+KARAF_DEBUG=<% @enable_java_debug %>
```

Comment:
this will get set to an empty string if java debugging is not enabled, "1" or "true" otherwise, depending on how it was set in the cluster config